ding/envs/env_wrappers/env_wrappers.py

```diff
@@ -616,6 +616,45 @@ def reset(self, **kwargs):
         return self.observation(observation)
 
 
+@ENV_WRAPPER_REGISTRY.register('obs_offline_norm')
+class ObsNormOfflineWrapper(gym.ObservationWrapper):
```

Comment:
Because the observation is normalized according to the statistics in the offline dataset. The potential candidate can be `ObsNormStaticWrapper` or `ObsNormFromDatasetWrapper`.